scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -35,6 +35,16 @@ test-dependency-rules:
   script:
     - ./scripts/ci/gitlab/ensure-deps.sh
 
+test-rust-features:
+  stage:                           check
+  extends:
+    - .kubernetes-env
+    - .test-refs-no-trigger-prs-only
+  variables:
+    CI_IMAGE:                      "paritytech/tools:latest"
+  script:
+    - ./scripts/ci/gitlab/rust-features.sh
```

Comment:
@paritytech/ci how do i install https://github.com/pksunkara/cargo-workspaces here as dependency for the script?

Reply:
In this case another `CI_IMAGE` should be used: `paritytech/ci-linux:production`